.vitepress/config.mts

```diff
@@ -49,6 +49,14 @@ export default defineVersionedConfig(
 
     themeConfig: {
       search: {
+        options: {
```

Comment:
This should be done using the `exclude` option instead.

https://github.com/vuejs/vitepress/pull/2602